cms/models.py

```diff
@@ -255,6 +266,76 @@ class FacultyMembersPage(CourseProgramChildPage):
     ]
 
 
+class HomePage(Page):
```

Comment:
just a question: I believe, we do not need to specify template_name to `home_page.html` in that class as that would the default convention for home page in wagtail. right ?

Reply:
No, you're right, that's the default conventions so we don't need to specify any template name.